crates/subspace-farmer/src/piece_cache.rs

```diff
@@ -303,21 +305,36 @@ where
 
         info!("Synchronizing piece cache");
 
-        // TODO: Query from the DSN too such that we don't build outdated cache at start if node is
```

Comment:
Is this `TODO` no longer apply here?

Reply:
Yes, we query it from the node instead now. I mean we could query DSN from farmer as well, but for simplicity of the implementation I decided to not do that